demos/python_demos/action_recognition/README.md

```diff
@@ -45,32 +46,33 @@ Options:
   -h, --help            Show this help message and exit.
   -i INPUT, --input INPUT
                         Required. An input to process. The input must be a single image,
-                        a folder of images or anything that cv2.VideoCapture can process
-  --loop                Optional. Enable reading the input in a loop
+                        a folder of images or anything that cv2.VideoCapture can process.
```

Comment:
```suggestion
                        a folder of images, video file or camera id.
```

Reply:
We need take into account that users might be not an OpenCV experts and so, not aware of what cv2.VideoCapture can process and how to supply input for VideoCapture. All the details should not be put into command line option description, seems that might be a separate common section in each demo readme. Need think on this. For this particular case, I think we may just change phrase a bit.

Reply:
done.